renjin/org/renjin/cran/stringi/stringi.java

```diff
@@ -111,7 +138,55 @@ public static void R_init_stringi(DllInfo dll) {
   public static SEXP stri_join_list(SEXP s1, SEXP s2, SEXP s3) { throw new EvalException("TODO"); }
   public static SEXP stri_join2(SEXP s1, SEXP s2) { throw new EvalException("TODO"); }
   public static SEXP stri_length(SEXP s1) { throw new EvalException("TODO"); }
-  public static SEXP stri_list2matrix(SEXP s1, SEXP s2, SEXP s3, SEXP s4) { throw new EvalException("TODO"); }
+  public static SEXP stri_list2matrix(SEXP x, SEXP byrow, SEXP fill, SEXP n_min) {
+    final boolean bycolumn = !byrow.getElementAsSEXP(0).asLogical().toBooleanStrict();
+    final String filler = fill.getElementAsSEXP(0).asString();
+    final int depth = n_min.getElementAsSEXP(0).asInt();
+    final int length = x.length();
+
+    if (depth < 0) {
+      // TODO log error message: "argument `n_min`: expected a nonnegative numeric value"
```

Comment:
You can throw an R error message with:

    throw new EvalException("error message");

Or add a warning with:

    Native.currentContext().warn("regexp engine config failed");